index.html

```diff
@@ -1,14 +1,101 @@
+        <h4 class="border-end flex-fill">Experts</h4>
+        <h4 class="ps-3">Vantage</h4>
+        <div class="search flex-fill">
+          <h6>Search <i class="fa fa-search"></i></h6>
+        </div>
+      </nav>
+      <nav class="d-none d-lg-flex flex-row nav-2 nav-3">
+        <h4 class="p-2 ps-3">TRENDING</h4>
+        <h4 class="p-2 ps-3 text-muted">JOE BIDEN</h4>
+        <h4 class="p-2 ps-3 text-muted">DONALD TRUMP</h4>
+        <h4 class="p-2 ps-3 text-muted">INAUGURATION</h4>
+        <h4 class="p-2 ps-3 text-muted">CORONAVIRUS</h4>
+        <h4 class="p-2 ps-3 text-muted">STIMULUS PACKAGE</h4>
+        <h4 class="p-2 ps-3 text-muted">VACCINE</h4>
+        <h4 class="p-2 ps-3 text-muted">CAPITOL RIOTS</h4>
+      </nav>
+    </header>
+    <main></main>
+    <footer class="ps-4 pe-4 pb-3">
+      <div class="border-bottom pt-5 pb-2 d-md-flex justify-content-between align-content-center">
+        <div class="d-md-flex">
+          <img class="brand-footer" src="img/newsweek.png" alt="">
+          <p class="copyright">© 2021 NEWSWEEK DIGITAL LLC</p>
+        </div>
+        <div class="social-icons d-flex">
+          <i class="fa fa-facebook-square"></i>
+          <i class="fa fa-twitter"></i>
+          <i class="fa fa-instagram"></i>
+          <i class="fa fa-tumblr"></i>
+          <i class="fa fa-linkedin"></i>
+        </div>
+      </div>
+
+      <div class="row d-flex">
+        <div class="d-flex">
+          <h6 class="mt-1 mb-1 ps-2 pe-2">Editions:</h6>
+          <a href="#" class="mt-1 mb-1"><span class="border-md-end ps-2 pe-2">U.S. Edition</span></a>
+          <a href="#" class="mt-1 mb-1"><span class="border-md-end ps-2 pe-2">Pakistan</span></a>
+          <a href="#" class="mt-1 mb-1"><span class="border-md-end ps-2 pe-2">Polska</span></a>
+          <a href="#" class="mt-1 mb-1"><span class="border-md-end ps-2 pe-2">Romania</span></a>
+        </div>
+        <div class="d-flex last-nav align-content-center">
+          <a href="#" class="flex-fill border-end"><span class="">About Us</span></a>
+          <a href="#" class="flex-fill border-end"><span>Corrections</span></a>
+          <a href="#" class="flex-fill border-end"><span>Contact Us</span></a>
+          <a href="#" class="flex-fill border-end"><span>Editorial Guideline</span></a>
+          <a href="#" class="flex-fill border-end"><span>Advertise</span></a>
+          <a href="#" class="flex-fill border-end"><span>Copyright</span></a>
+          <a href="#" class="flex-fill border-end"><span>Terms & Conditions</span></a>
+          <a href="#" class="flex-fill border-end"><span>Privacy</span></a>
+          <a href="#" class="flex-fill border-end"><span>Cookies Policy</span></a>
+          <a href="#" class="flex-fill border-end"><span>Terms of Sale</span></a>
+          <a href="#" class="flex-fill border-end"><span>Archive</span></a>
+          <a href="#" class="flex-fill"><span>Announcement</span></a>
+        </div>
+      </div>
```

Comment:
- [ ] Alignment of this item is wrong on mobile view. On the original page, these items aligned to left. Please check the attached screenshots. 
 -------------
 **Your Version:**
![Image](https://imgur.com/download/dNXCGFn/)

--------------
**Original:**
![Image2](https://imgur.com/download/e9c3aIy/)